### PR TITLE
Add Xamarin android default app run to maui_android runs.

### LINKED ISSUE #3233: Measure performance of default .NET Android template
- The `helloandroid` scenario measures a raw Android Hello World app from `dotnet/runtime`
- The `mauiandroid` scenario measures a MAUI Android app from `dotnet new maui`

It would be nice for the .NET Android team if there was a scenario between those 2 that measures the `dotnet new android` scenario.  

This would allow us to monitor the performance of the code in https://github.com/xamarin/xamarin-android without bringing in the MAUI specific bits, which would help us detect regressions specific to `xamarin-android`.

This test could copy everything from the `mauiandroid` scenario, but change the `dotnet new` template from `maui` to `android`.

### PR DESCRIPTION
**Pull request by LoopedBard3**
Closes #3233. 

Need to verify the version file we should be using, the tests themselves appear to be successful.
Test run: https://dev.azure.com/dnceng/internal/_build/results?buildId=2261386&view=results

### CODE REVIEW

**Comment by LoopedBard3** [comment]
cc: @akoeplinger @kotlarmilos Do either of y'all know what the correct file to get the XamarinVersion from is or what file should we get what version(s) from?

**Review by akoeplinger — DISMISSED** [review]
LGTM with one naming comment. Getting the version from Mono.Android.dll should be fine.

File: eng/performance/maui_scenarios_android.proj [hunk 1]
```diff
@@ -33,6 +33,12 @@
       <ApkName>com.companyname.mauiandroiddefault-Signed</ApkName>
       <PackageName>com.companyname.mauiandroiddefault</PackageName>
     </MAUIAndroidScenario>
+    <MAUIAndroidScenario Include="Xamarin Android Default">
```
[the remark this answers is not in the document]
Comment (akoeplinger) [reply eng/performance/maui_scenarios_android.proj:36]: this is still unaddressed.

**Review by kotlarmilos — COMMENTED** [review]
Apart from the comment above, LGTM!

> The helloandroid scenario measures a raw Android Hello World app from dotnet/runtime

Since this PR closes the tracking issue, are the measurements for the hello world app already implemented?

File: eng/performance/maui_scenarios_android.proj (hunk 1 above)
[the remark this answers is not in the document]
Comment (LoopedBard3) [reply eng/performance/maui_scenarios_android.proj]: Updated to the name '.NET Android Default'

**Comment by LoopedBard3** [comment]
> Since this PR closes the tracking issue, are the measurements for the hello world app already implemented?

We are already running the measurements for the hello world android and ios apps in runtime.

Android scenario file: https://github.com/dotnet/runtime/blob/main/eng/testing/performance/android_scenarios.proj

Scenario top level definition: https://github.com/dotnet/runtime/blob/1e5ee2dc46508223f1bb04fc21a8003cc7605442/eng/pipelines/coreclr/perf-non-wasm-jobs.yml#L137-L150

Build specification location (same file as run but above the run): https://github.com/dotnet/runtime/blob/1e5ee2dc46508223f1bb04fc21a8003cc7605442/eng/pipelines/coreclr/perf-non-wasm-jobs.yml#L62-L81

Second level build with make commands: https://github.com/dotnet/runtime/blob/1e5ee2dc46508223f1bb04fc21a8003cc7605442/eng/pipelines/coreclr/templates/build-perf-sample-apps.yml#L22-L37

File: src/scenarios/netandroid/pre.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from shared.precommands import PreCommands
+from shared.versionmanager import versions_write_json, get_version_from_dll_powershell
+from test import EXENAME
+
+setup_loggers(True)
+
+precommands = PreCommands()
+install_versioned_maui(precommands)
+
+# Setup the Maui folder
```
Comment (cincuranet) [inline src/scenarios/netandroid/pre.py:18]: nit: Is it really MAUI folder?

**Review by cincuranet — APPROVED** [review]

File: src/scenarios/netandroid/pre.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+precommands.execute([])
+
+# Remove the aab files as we don't need them, this saves space
+output_dir = const.PUBDIR
+if precommands.output:
+    output_dir = precommands.output
+remove_aab_files(output_dir)
+
+# Copy the xamarinVersion to a file so we have it on the machine
+xamarin_version = get_version_from_dll_powershell(rf".\{const.APPDIR}\obj\Release\{precommands.framework}\android-arm64\linked\Mono.Android.dll")
```
Comment (akoeplinger) [inline src/scenarios/netandroid/pre.py:37]: nit: might want to call this netandroid_version too to avoid confusion

**Review by akoeplinger — APPROVED** [review]

File: src/scenarios/netandroid/pre.py (hunk 3 above)
Comment (LoopedBard3) [reply src/scenarios/netandroid/pre.py:37]: Updated to net_android

**Review by cincuranet — APPROVED** [review]

**Comment by LoopedBard3** [comment]
Failures are unrelated and test run was successful.

**Pull request merged**